File: back_end/internal/usecase/dto/login_dto.go
package dto

type InputLoginDto struct {
	Email    string `json:"email"`
	Password string `json:"password"`
}
type InputLogoutDto struct {
	SessionToken string `json:"session_token"`
}

type OutPutLoginDto struct {
	Mensage string `json:"mensage"`
}

type OutPutLoginTokenDto struct {
	UserId string `json:"user_id"`
}

File: front_end/services/dto/login_dto.ts
export interface CreateLoginInputDto {
    email: string
    password: string
}

export interface OutputLogintDto {
   mensage:string
   user_id:string
 
}
export interface DeleteLogintInputDto {
    id: string
}
export interface OutputDeleteLogintDto {
    mensage: string
 
}